diff --git a/client/src/external/daemon/runner.py b/client/src/external/daemon/runner.py
index 6973cf1..a34d9cb 100644
--- a/client/src/external/daemon/runner.py
+++ b/client/src/external/daemon/runner.py
@@ -108,9 +108,9 @@ class DaemonRunner:
         self.app = app
         self.daemon_context = DaemonContext()
         self.daemon_context.stdin = open(app.stdin_path, 'rt')
-        self.daemon_context.stdout = open(app.stdout_path, 'w+t')
+        self.daemon_context.stdout = open(app.stdout_path, 'a+t')
         self.daemon_context.stderr = open(
-                app.stderr_path, 'w+t', buffering=0)
+                app.stderr_path, 'a+t', buffering=0)

         self.pidfile = None
         if app.pidfile_path is not None:
